### ISSUE TITLE
Fallback config invalid JSON crashes launch

### ISSUE THREAD

**Issue by FunFreshNew** [ZsoltMolnarrr/BetterCombat#197]
**Minecraft version** - 1.19.2
**Mod loader** - Forge
**Mod version** - 43.2.0
**I am using the latest version available** - Yes, for the latest 1.19.2

**Describe the bug**
Can't load mods and keep showing this
"Error loading mods"
Better Combat (bettercombat) has failed to load correctly
java.lang.reflect.InvocationTargetException: null

**To Reproduce**
Previously, the Minecraft run smoothly and nothing wrong with it. But somehow the game crashed

When I tried to re-open minecraft, the game won't open and keep showing Error notice 

**Expected behavior**
Basically run minecraft normally.

**Screenshots or video recordings**
![image](https://user-images.githubusercontent.com/13015615/216107294-60e3a9fa-3691-4edc-9ec6-7a75e2fdd9cb.png)

**What I already tried**
I upgrade/downgrade forge, clean "version" folder

**Additional context**
The FML https://pastebin.com/yqFuQiKL

The latest log https://pastebin.com/wBRn7VLR

**Comment by ZsoltMolnarrr** [comment]
Tips to resolve **Forge** specific issues:

**:boom: Game crashes upon launch**
Install all other required mods. These are listed on the page of the downloadable file itself.
(Forge fails to show missing dependencies.)

**:ghost: Mod doesn't show up in Forge mod menu**
Use the __recommended__ version of Forge __and not the latest__. You can check it here: 
(This is a random issue of Forge, some versions work correctly, some don't.)
https://files.minecraftforge.net/net/minecraftforge/forge/

**Mod fails to be unzipped**
This is an error of Forge mod loader. I have no idea why it happens, try a different version of the mod loader.
(This is a random issue of Forge, some versions work correctly, some don't.)

**:snail: Animations lagging**
Replace Optifine with an open source alternative.

**Comment by ZsoltMolnarrr** [comment]
Are you sure your Cloth Config version is `cloth-config-8.2.88-forge` ?

**Comment by FunFreshNew** [comment]
> Are you sure your Cloth Config version is `cloth-config-8.2.88-forge` ?

yea I have that installed

**Comment by ZsoltMolnarrr** [comment]
Please try deleting the fallback configuration file, or disable using fallback compatibility in server config.

**Comment by FunFreshNew** [comment]
> Please try deleting the fallback configuration file, or disable using fallback compatibility in server config.

YOOOOOOOOOO it works!

**Comment by ZsoltMolnarrr** [comment]
The fallback configuration got most likely corrupted as a result of editing.
This is a bug I need to fix.

**Comment by FunFreshNew** [comment]
> The fallback configuration got most likely corrupted as a result of editing. This is a bug I need to fix.

yea It was my speculation at first but no idea where the file is

**Comment by ZsoltMolnarrr** [comment]
I cannot reproduce the issue.
As soon as someone can do so, feel free to reopen the issue and attach the faulty config file
